.php-cs-fixer.dist.php

```diff
@@ -12,68 +12,81 @@
+    'single_quote' => ['strings_containing_single_quote_chars' => true],
+    'visibility_required' => ['elements' => ['property', 'method', 'const']],
+    'yoda_style' => ['equal' => false, 'identical' => false, 'less_and_greater' => false],
 
-            // @Symfony customised rules
-            'concat_space' => ['spacing' => 'one'],
-            'native_function_invocation' => false,
-            'single_quote' => ['strings_containing_single_quote_chars' => true],
-            'visibility_required' => ['elements' => ['property', 'method', 'const']],
-            'yoda_style' => ['equal' => false, 'identical' => false, 'less_and_greater' => false],
+    // Additional code style rules whitelisting:
+    'align_multiline_comment' => true,
+    'array_indentation' => true,
+    'array_syntax' => ['syntax' => 'short'],
+    'combine_consecutive_issets' => true,
+    'declare_strict_types' => true,
+    'explicit_indirect_variable' => true,
+    'explicit_string_variable' => true,
+    'fully_qualified_strict_types' => true,
+    'linebreak_after_opening_tag' => true,
+    'list_syntax' => ['syntax' => 'short'],
+    'mb_str_functions' => false,
+    'multiline_comment_opening_closing' => true,
+    'multiline_whitespace_before_semicolons' => ['strategy' => 'new_line_for_chained_calls'],
+    'no_alternative_syntax' => true,
+    'no_superfluous_elseif' => true,
+    'ordered_imports' => true,
+    'ordered_interfaces' => true,
+    'no_superfluous_phpdoc_tags' => ['allow_mixed' => true,'remove_inheritdoc' => false],
+];
+$version = explode('.', PHP_VERSION);
+if ($version[0] > 7) {
+    // a supprimer en monorepo v2 uniquement >=PHP 8.0
+    // @todo a supprimer en monorepo v2 uniquement >=PHP 8.0
+    $rules += [ "get_class_to_class_keyword" => false ];
+}
 
-            // Additional code style rules whitelisting:
-            'align_multiline_comment' => true,
-            'array_indentation' => true,
-            'array_syntax' => ['syntax' => 'short'],
-            'combine_consecutive_issets' => true,
-            'declare_strict_types' => true,
-            'explicit_indirect_variable' => true,
-            'explicit_string_variable' => true,
-            'fully_qualified_strict_types' => true,
-            'global_namespace_import' => ['import_classes' => false, 'import_constants' => false, 'import_functions' => false],
-            'list_syntax' => ['syntax' => 'short'],
-            'mb_str_functions' => true,
-            'multiline_comment_opening_closing' => true,
-            'multiline_whitespace_before_semicolons' => ['strategy' => 'new_line_for_chained_calls'],
-            'no_alternative_syntax' => true,
-            'no_superfluous_elseif' => true,
-            'ordered_interfaces' => true,
-        ]
-    )
-    ->setRiskyAllowed(true)
+$config = new PhpCsFixer\Config();
+$config->setRiskyAllowed(true)
+    ->setRules($rules)
     ->setFinder($finder)
-    ->setCacheFile('.php-cs-fixer.cache')
+    ->setUsingCache(true)
```

Comment:
Le cache est activé par défaut : https://cs.symfony.com/doc/usage.html#caching

```suggestion
    ->setCacheFile('.php-cs-fixer.cache')
```